packages/login/src/vaadin-login-form.js

```diff
@@ -165,24 +167,14 @@ class LoginForm extends LoginMixin(ElementMixin(ThemableMixin(PolymerElement)))
     }
   }
 
-  /** @private */
-  __isValid(input) {
-    return (input.validate && input.validate()) || (input.checkValidity && input.checkValidity());
```

Comment:
Simplified this logic, as both `vaadin-text-field` and `vaadin-password-field` support `validate()`.